typescript/infra/chain-config.json

```diff
@@ -0,0 +1,1429 @@
+{
```

Comment:
did you mean to check this in? seems like a mostly dupe of `rust/sealevel/environments/mainnet3/chain-config.json`